otx/algorithms/detection/adapters/mmdet/tasks/trainer.py

```diff
@@ -103,7 +107,7 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         # cfg.dump(osp.join(cfg.work_dir, 'config.py'))
         # logger.info(f'Config:\n{cfg.pretty_text}')
 
-        validate = True if cfg.data.get("val", None) else False
+        validate = cfg.data.get("val", None)
```

Comment:
```suggestion
        validate = "val" in cfg.data
```
validate should be boolean type